libs/libvqm/vqm_parser.l

```diff
@@ -38,7 +38,9 @@
 
 %%
 ^[ \t]*\/\/[^\n\r]*(\n|\r\n)                /* skip one-line comments */
+^[ \t]*\(\*[^\n\r]*\*\)                     /* skip synthesis attributes and directives */
 [ \t]+                                      /* skip white spaces */
+!					    /* skip the logical operator ! applied on the input ports of the lut - this results in lut mask not being valid anoymore */
```

Comment:
probably want to explain this means inversion in the vqm netlist, but we are ignoring it (hence the LUTmask won't be correct).